tests/integration/build_invoke/test_build_invoke_go1_x.py

```diff
@@ -15,6 +16,14 @@ class BuildInvoke_go1_x_cookiecutter_aws_sam_hello_golang(BuildInvokeBase.BuildI
     use_container = False
     directory = "go1.x/cookiecutter-aws-sam-hello-golang"
 
+class BuildInvoke_go1_x_cookiecutter_aws_sam_eventbridge_hello_golang(BuildInvokeBase.BuildInvokeBase):
+    use_container = False
+    directory = "go1.x/cookiecutter-aws-sam-eventbridge-hello-golang"
+
+@skip("eventbridge schema app requires credential to pull missing files, skip")
```

Comment:
could you please explain why this sample requires credential. I could not see any logic in `main.go` that it requires credential.

Reply:
I got it, the retrieval step done during `sam init` as it is a dynamic template.

Reply:
is it ok to remove this test case, it will not be used anyway, and add it to sam cli instead.

Reply:
Hmmm .. it is ok, but can you add todo comment to remove the skip once we start providing

Reply:
Todo added to unskip tests when they are run in an environment with AWS Credentials